classes.py

```diff
@@ -0,0 +1,544 @@
+                    else:
+                        raise ValueError('Wrong tournament type: %s' % trn_type)
+
+                    tournament.complete_players_info()
+                    tournament.calculate_players_ratings()
+                    tournament.write_new_ratings_list(final_rating_filepath)
+
+                initial_rating_filepath = final_rating_filepath
+
+        self.write_manual_entry_dict()
+
+    def get_rating_list(self, initial_rating_filepath):
+
+        with open(initial_rating_filepath) as rating_list:
+            reader = csv.reader(rating_list, delimiter=_CSV_DELIMITER)
+            next(reader, None)  # Skip the headers
+            for row in reader:
+                self.rating_list.update({int(row[0]): FexerjPlayer(int(row[0]),  # ID FEXERJ
+                                                                   row[1],  # ID CBX
+                                                                   row[2],  # TITLE
+                                                                   row[3],  # NAME
+                                                                   int(row[4]),  # RATING
+                                                                   row[5],  # CLUB
+                                                                   row[6],  # BIRTHDAY
+                                                                   row[7],  # SEX
+                                                                   row[8],  # FEDERATION
+                                                                   int(row[9]),  # TOTAL NUM OF GAMES
+                                                                   row[10],  # SUM OF OPPONENT RATINGS
+                                                                   row[11])})  # POINTS AGAINST OPPONENTS
+                if len(row[1]) > 0:
+                    self.cbx_to_fexerj[int(row[1])] = int(row[0])
+
+    def load_manual_entry_dict(self):
+        if os.path.exists('manual_entry_list.json'):
+            with open('manual_entry_list.json', 'r') as manual_entry_dict_file:
+                self.manual_entries = json.load(manual_entry_dict_file)
+
+    def write_manual_entry_dict(self):
+        with open('manual_entry_list.json', 'w') as manual_entry_dict_file:
+            manual_entry_dict_file.write(json.dumps(self.manual_entries))
+
+
+class FexerjPlayer:
+
+    def __init__(self, id_fexerj, id_cbx, title, name, lastrating, club, birthday, sex, federation, totgames,
+                 sumopprating, ptsagainstopp):
+        self.id_fexerj = id_fexerj
+        self.id_cbx = id_cbx
+        self.title = title
+        self.name = name
+        self.lastRating = lastrating
+        self.club = club
+        self.birthday = birthday
+        self.sex = sex
+        self.federation = federation
+        self.totGames = totgames
+        self.sumOppRating = sumopprating
+        self.ptsAgainstOpp = ptsagainstopp
+
+
+class TournamentPlayer:
+
+    def __init__(self, tournament, url):
```

Comment:
(nit) I'd rename url to player_url, to make it clearer what this is. Maybe add some pydoc string with a link to an example.